src/mender-update/cli/cli.cpp

```diff
@@ -57,6 +57,18 @@ const conf::CliCommand cmd_daemon {
 	.description = "Start the client as a background service",
 };
 
+const conf::CliOption opt_stop_after {
+	conf::CliOption {
+		.long_option = "stop-after",
+		.description =
+			"Stop after the given state has completed. "
+			"Choices are `Download`, `ArtifactInstall` and `ArtifactCommit`. "
+			"You can later resume the installation by using the `resume` command. "
+			"Note that the client always stops after `ArtifactInstall` if the update module supports rollback.",
```

Comment:
and/or if the update module requested a reboot, right?

Reply:
No, see [earlier comment](https://github.com/mendersoftware/mender/pull/1642#discussion_r1706745507).